Hestia/Hestia/frontend/Local/UITableViewControllerServerConnect.cs

```diff
@@ -93,43 +98,32 @@ public override bool ShouldPerformSegue(string segueIdentifier, NSObject sender)
 
             try
             {
-                validIp = PingServer.Check(newIP.Text, int.Parse(newPort.Text));
+                validIp = PingServer.Check(strings.defaultPrefix + newIP.Text, int.Parse(strings.defaultPort));
             }
             catch (Exception exception)
             {
                 Console.WriteLine(exception.StackTrace);
-                DisplayWarningMessage();
+                new WarningMessage("Could not connect to server", "Invalid server information", this);
+                connectButton.Selected = false;
                 return false;
             }
 
             if (validIp)
             {
-                userDefaults.SetString(newServerName.Text, strings.defaultsServerNameHestia);
-                userDefaults.SetString(newIP.Text, strings.defaultsIpHestia);
-                userDefaults.SetString(newPort.Text, strings.defaultsPortHestia);
-
                 Globals.ServerName = newServerName.Text;
-                Globals.IP = newIP.Text;
-                Globals.Port = int.Parse(newPort.Text);
-                HestiaServerInteractor serverInteractor = new HestiaServerInteractor(new NetworkHandler(Globals.IP, Globals.Port));
+                Globals.Address = strings.defaultPrefix + newIP.Text;
+                HestiaServerInteractor serverInteractor = new HestiaServerInteractor(new NetworkHandler(Globals.Address, int.Parse(strings.defaultPort)));
```

Comment:
As you are always passing default port, you could just remove it and put that in the networkHandler

Reply:
The NetworkHandler is also used to interact with the web teams server, that's why we can't add the port by default in the NetworkHandler. As I already noticed the port after #223 contained in the address.